ci-operator/step-registry/ipi/install/ipi-install-chain.yaml

```diff
@@ -6,5 +6,7 @@ chain:
   - ref: ipi-install-hosted-loki
   - ref: ipi-install-install
   - ref: ipi-install-times-collection
+  - ref: nodes-readiness
+  - ref: multiarch-validate-nodes
```

Comment:
Per the discussion in https://github.com/openshift/release/pull/54703#discussion_r1695388886, we are going to merge the two steps into one later, right?

Reply:
I would actually prefer keeping them separate. I like two simpler steps as they are mostly doing two different things.

Reply:
Let's follow up on https://github.com/openshift/release/pull/54703#discussion_r1698870796

I would wrap this up while we try to get all in the same page.